frame/staking/src/lib.rs

```diff
@@ -1284,6 +1286,17 @@ decl_module! {
 			ForceEra::put(Forcing::ForceNew);
 		}
 
+		/// Force there to be a new era at the end of sessions indefinitely.
```

Comment:
I had a similar dilemma of wanting to re-order all of these but I ended up not doing it since it might mess with ongoing proposals with changing the call index. Should make sure that won't happen now. We usually have `staking.setValidatorCount` in the live proposals.

Reply:
ah that is a good point. Can revert these changes if needed